state/charm.go

```diff
@@ -251,7 +251,7 @@ func insertCharmOps(mb modelBackend, info CharmInfo) ([]txn.Op, error) {
 	}
 	lpc, ok := info.Charm.(charm.LXDProfiler)
```

Comment:
Why didn't a test get added that exercises this?

Reply:
Added a compile time check